nxdrive/engine/engine.py

```diff
@@ -448,12 +450,20 @@ def _save_last_dt_session_infos(
             )
 
     def _create_remote_folder(
-        self, remote_parent_path: str, new_folder: str, /
+        self, remote_parent_path: str, new_folder: str, session_id: int, /
     ) -> Dict[str, Any]:
         try:
             return self.remote.upload_folder(
                 remote_parent_path,
                 {"title": new_folder},
+                headers={
+                    REQUEST_METRICS: json.dumps(
+                        {
+                            DT_NEW_FOLDER: 1,
+                            DT_SESSION_NUMBER: session_id,
+                        }
+                    )
+                },
```

Comment:
Reading that line again, and now I have a big doubt: is it required to use `json.dump()` ourselves? Maybe `requests` will do it anyway on the whole `header`?

Reply:
nope, it will trigger an error